annotation-server/README.md

```diff
@@ -3,8 +3,10 @@
 ## Swagger Endpoint Documentation (OpenAPI)
 
 You can find all available API endpoints defined as Swagger Documentation
-[here](http://localhost:3000/api-docs) (currently just when you run the sever on
-`localhost`, later on available through the VM).
+[here](http://172.20.24.66:8080/api).
+
+> Please note that this endpoint is only available in the HPI network.
+> That means you either need to be connected directly or through a VPN.
```

Comment:
```suggestion
[here](http://172.20.24.66:8080/api-docs).

> Please note that this endpoint is only available in the HPI network.
> That means you either need to be connected directly or through a VPN.
```

Currently, the route to the swagger documentation is root/api-docs not root/api. We can change that though if you like.

Reply:
As written in the PR description this PR depends on #182 which changes exactly that (`api-docs` ➞ `api`)